tensorflow/lite/delegates/gpu/common/tasks/fully_connected_test_util.cc

```diff
@@ -53,7 +53,6 @@ absl::Status FullyConnectedTest(TestExecutionEnvironment* env) {
           src_tensor, std::make_unique<FullyConnected>(std::move(operation)),
           BHWC(1, 1, 1, 2), &dst_tensor));
       RETURN_IF_ERROR(PointWiseNear({14.5f, 37.5f}, dst_tensor.data, eps))
-          << "Failed using precision " << ToString(precision);
```

Comment:
why?

Reply:
I get compile errors since RETURN_IF_ERROR() doesn't return an ostream. I can't root-cause from the history how it worked though. I assume it might be related to the abseil status changes.